components/script/dom/window.rs

```diff
@@ -2102,8 +2109,12 @@ impl Window {
         self.Document().url()
     }
 
-    pub fn layout_chan(&self) -> &Sender<Msg> {
-        &self.layout_chan
+    pub fn layout_chan(&self) -> Option<&Sender<Msg>> {
+        if self.is_alive() {
+            Some(&self.layout_chan)
+        } else {
+            None
+        }
     }
```

Comment:
This could be simplified to:

```diff
     pub fn layout_chan(&self) -> Option<&Sender<Msg>> {
-        if self.is_alive() {
-            Some(&self.layout_chan)
-        } else {
-            None
-        }
+        self.is_alive().then_some(&self.layout_chan)
     }
```

If we add the `#![feature(bool_to_option)]` to the crate, but I'm not sure what's the policy on nightly features is for servo :)

Reply:
We are open to nightly features if they significantly improve the code in some way, but I don't think this particular case warrants it :)